src/Expensify.js

```diff
@@ -1,42 +1,41 @@
-import _ from 'underscore';
 import lodashGet from 'lodash/get';
 import PropTypes from 'prop-types';
-import React, {useCallback, useState, useEffect, useRef, useLayoutEffect, useMemo} from 'react';
+import React, {useCallback, useEffect, useLayoutEffect, useMemo, useRef, useState} from 'react';
 import {AppState, Linking} from 'react-native';
 import Onyx, {withOnyx} from 'react-native-onyx';
+import _ from 'underscore';
+import ConfirmModal from './components/ConfirmModal';
```

Comment:
It seems this file is using relative imports still – I thought that the prefer-alias rule would disallow this?

Reply:
This is expected. Maybe because this path doesn't contain parent route but sub route (no `../` exists).

<img width="696" alt="Screenshot 2023-10-19 at 11 10 26 AM" src="https://github.com/Expensify/App/assets/108292595/a5961342-3efb-4a5b-8966-069c6a35335b">

Reply:
gotcha 👍🏼

Reply:
I actually kind of wish that we could configure this to not be the way it behaves, maybe we do that as a follow-up: https://github.com/dword-design/eslint-plugin-import-alias/issues/133

Reply:
https://expensify.slack.com/archives/C01GTK53T8Q/p1696838981418589?thread_ts=1696833372.663479&cid=C01GTK53T8Q maybe we use eslint for import order as well? but that import order does require some extra configuration.